packages/website/pages/account/payment.js

```diff
@@ -1,60 +1,35 @@
+// @ts-nocheck
```

Comment:
Hey this seems like cheating :) @cmunns Are you opposed to using type-checking in this file?